src/effects/VST3/VST3Utils.h

```diff
@@ -15,6 +15,9 @@
 #include <pluginterfaces/vst/ivstaudioprocessor.h>
 #include <string>
 #include <public.sdk/source/vst/hosting/module.h>
+#include <public.sdk/source/vst/vstpresetfile.h>
```

Comment:
This inclusion at the second commit lets you remove the header in one other place.

Reply:
Right, done